dll/cpl/mmsys/volume.c

```diff
@@ -45,54 +45,149 @@ typedef struct _GLOBAL_DATA
 
 
 static VOID
-InitImageInfo(PIMGINFO ImgInfo)
+InitImageInfo(HWND hwndDlg, PIMGINFO ImgInfo)
 {
     BITMAP bitmap;
+    BITMAP maskBitmap;
+    BITMAPINFO bmpi;
+    HDC hDC, hDCBitmap, hDCMask;
+    HBITMAP hBitmap = NULL, hMask = NULL;
+    HBITMAP hAlphaBitmap = NULL;
+    COLORREF *pBits;
+    INT line, column;
 
     ZeroMemory(ImgInfo, sizeof(*ImgInfo));
 
```

Comment:
What's the purpose of all the code that follows?

Reply:
Moved the fix to next PR.